### PR TITLE
Create course

### PR DESCRIPTION
**Pull request by Christen97**
I am trying to implement create courses in ScalableTeaching.
So far I have implemented:

- [x] View for create course
- [x] Route and controller method for create course
- [x] Feature test
- [x] Validation
- [x] Styling

### CODE REVIEW

**Comment by Christen97** [comment]
Styling draft:
CreateCourseST

**Comment by Christen97** [comment]
Styling draft:
CreateCourseST

**Comment by jazerix** [comment]
Looking good! Can you make the button white while it's not hovered? otherwise it's a go 😄

**Comment by jazerix** [comment]
When doing validation, you can look through this documentation: https://laravel.com/docs/9.x/validation#main-content

In general, you want to crate some rules that the name has to pass. You can see all rules available in the link above. Laravel will redirect back if validation fails, you just need to show the error to the user, see [Displaying The Validation Errors](https://laravel.com/docs/9.x/validation#quick-displaying-the-validation-errors)

When testing validation you can use tester methods like [assertSessionHasErrors](https://laravel.com/docs/9.x/http-tests#assert-session-has-errors)

File: app/Policies/CoursePolicy.php [hunk 1]
```diff
@@ -24,6 +24,11 @@ public function view(User $user, Course $course)
         return $course->users()->where('user_id', $user->id)->exists();
     }
 
+    public function create(User $user)
+    {
+        return false;
```
Comment (jazerix) [inline app/Policies/CoursePolicy.php:29]: Why is this set to false?

Comment (Christen97) [reply app/Policies/CoursePolicy.php:29]: Since the user is admin it will be set to true in `AuthServiceProvider`.

**Comment by jazerix** [comment]
It occurs to me that we need this to also create a gitlab group as part of the creation process

**Comment by mcamp** [comment]
Fine by me, if @jazerix is ok with it, solve the conflict and merge

**Comment by Christen97** [comment]
> It occurs to me that we need this to also create a gitlab group as part of the creation process

@jazerix I am not totally sure what you mean, is it possible you could elaborate a bit?

**Review by jazerix — APPROVED** [review]

**Pull request merged**